NavigationReact/sample/relay-modern/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "navigation-relay",
+  "description": "Navigation React Relay (modern) example",
+  "dependencies": {
+    "babel-cli": "^6.26.0",
+    "babel-core": "^6.26.0",
+    "babel-loader": "^7.1.2",
+    "babel-polyfill": "^6.26.0",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-react": "^6.24.1",
+    "babel-preset-stage-0": "^6.24.1",
+    "express": "^4.15.4",
+    "express-graphql": "^0.6.7",
+    "graphql": "^0.11.1",
+    "navigation": "*",
+    "navigation-react": "*",
+    "react": "*",
+    "react-dom": "*",
+    "react-relay": "^1.2.0",
+    "webpack": "^3.5.5"
+  },
+  "scripts": {
+    "start": "babel-node ./server.js",
+    "updateSchema": "babel-node ./updateSchema.js",
+    "relay": "relay-compiler --src . --schema ./schema.json"
+  },
+  "devDependencies": {
```

Comment:
I don’t use `devDependencies` in examples. I put everything under `dependencies`.

In the Relay example, `webpack` bundles at runtime on the server. This isn’t how it should be done for real but it simplifies the code. So should I put `webpack` in `devDependencies` or `dependencies`? To avoid this sort of confusion I put everything under `dependencies`.